moveit_experimental/collision_distance_field/include/moveit/collision_distance_field/collision_world_distance_field.h

```diff
@@ -140,6 +140,18 @@ class CollisionWorldDistanceField : public CollisionWorld
     return 0.0;
   }
 
+  virtual void distanceRobot(const DistanceRequest& req, DistanceResult& res,
+                             const CollisionRobot& robot, const robot_state::RobotState& state) const
+  {
+    logWarn("Not implemented");
```

Comment:
I think this should be `logError`.

Reply:
I agree, this is the way it was originally, but I am happy to update them.

Reply:
Not yet addressed.